docs/client-specifications/feature-matrix.md

```diff
@@ -0,0 +1,51 @@
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+# Momento Client SDKs: Feature Matrix
+
+This page should serve as a checklist for developers who are preparing the client library
+for a given language for a 1.0 release.
+
+## Features / Requirements
+
+* `Release`: Repo has been set up according to our [SDK release process](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-sdk-release-process.md) and binary artifacts are published to the appropriate
+             package repository for the language.
+* `Get/Set APIs`: Support for basic get/set APIs, with Strings or byte arrays.
```

Comment:
Also cache create, delete, and list.

Note that AlreadyExists is not an error from CreateCache.